index.js

```diff
@@ -378,21 +330,23 @@ var ModalBox = createReactClass({
             return <View />
         }
         return (
-            <View style={[styles.transparent, styles.absolute]} pointerEvents={'box-none'} onLayout={this.onContainerLayout}>
-                {backdrop}
-                <Animated.View
-                    onLayout={this.onViewLayout}
-                    style={[styles.wrapper, size, this.props.style, { transform: [{ translateY: this.state.position }, { translateX: offsetX }] }, Platform.OS === 'web' ? { willChange: 'transform' } : {}]}
-                    {...this.state.pan.panHandlers}>
-                    {this.props.children}
-                </Animated.View>
-            </View>
+            <Portal>
```

Comment:
How will Portal work for android and iOS?

Reply:
This is only for web, not used by android or ios

Reply:
Added a check for portals using prop showFullscreen